packages/react/src/components/actions/Action.tsx

```diff
@@ -24,7 +24,7 @@ export interface IBaseActionOptions {
     target?: string;
     primary?: boolean;
     tooltip?: string;
-    tooltipPlacement?: string;
+    tooltipPlacement?: TooltipPlacement;
```

Comment:
That's actually breaking if people were using the string value (e.g., `tooltipPlacement='left'` )

Could we just use `'left' | 'right' | 'top' | 'bottom'` ? (I'm not sure we can generate this from the Typescript enum)

```suggestion
    tooltipPlacement?: 'left' | 'right' | 'top' | 'bottom';
```